Vagrantfile~

```diff
@@ -0,0 +1,41 @@
+Vagrant.configure('2') do |config|
+
+  # Configure the box to use
+    config.vm.box='ubuntu/trusty64'
+      config.vm.define "jens_virtualmachine" do |foohost|
+      end
+      config.vm.provider :virtualbox do |vb|
+        vb.name = "exercism_virtualbox"
+      end
+  # If you can only use a 32-bit operating system, Comment line 3 and uncomment lines 5
+    # config.vm.box='ubuntu/trusty32'
+
+  # Configure the network interfaces
+    config.vm.network :private_network, ip:    "33.33.33.33"
```

Comment:
It's a good idea to ensure your private network ip addresses are always in a reserved private network range.
https://en.wikipedia.org/wiki/Reserved_IP_addresses

So "10.33.33.33" would be a better number.